### PR TITLE
find secure route dynamically

### PR DESCRIPTION
**Pull request by philbrookes**
## Motivation
INTLY-458

## Verification:
https://master.pbrookes-b14d.openshiftworkshop.com/console

evals01 @ https://tutorial-web-app-webapp.apps.pbrookes-b14d.openshiftworkshop.com/
evals01 @ http://launcher-launcher.apps.pbrookes-b14d.openshiftworkshop.com/launch/

### CODE REVIEW

File: evals/roles/customisation/tasks/main.yaml [hunk 1]
```diff
@@ -94,9 +94,13 @@
     components: "{{components}} + [{{che_manifest}}] "
   when: che and launcher
 
+- name: Find encrypted RH-SSO route
+  shell: for route in $(oc get routes -n {{ rhsso_namespace}} | awk '{print $1}' | grep -v 'NAME'); do term=$(oc get route $route -n {{ rhsso_namespace }} -o template --template \{\{.spec.tls.termination\}\}); if [ "$term" == "edge" ] || [ "$term" == "reencrypt" ]; then echo $route; break; fi; done
```
Comment (maleck13) [inline evals/roles/customisation/tasks/main.yaml]: wow seems unfortunate that this is how we have to find a secure route. Don't suppose there are annotations added to the route or labels?

**Comment by maleck13** [comment]
is there anything we can do about the repetitive nature of this?

File: evals/roles/customisation/tasks/main.yaml (hunk 1 above)
Comment (philbrookes) [reply evals/roles/customisation/tasks/main.yaml]: yeah, I had a look at that first, didn't see anything I felt comfortable relying on, like "secure: true"

**Comment by philbrookes** [comment]
@maleck13 we could possibly make a python library that does this and call to that? Want me to start working on that now? We can merge this to unblock the release in the mean time?

**Comment by JameelB** [comment]
the launcher sso route is also being used here: https://github.com/integr8ly/installation/blob/master/evals/playbooks/install_services.yml#L55 I think the Che deployment fails if it also doesn't get the correct launcher sso route.

**Comment by philbrookes** [comment]
@jameelb oh thanks, I did overlook that, strange that it still worked for me then. I'll update it now.

**Comment by philbrookes** [comment]
@JameelB this is updated now, and re-provisioned to the same cluster for verification.

**Comment by maleck13** [comment]
verifying on my cluster

**Comment by maleck13** [comment]
was able to login to the webapp

**Pull request merged**